genai/skaffold.yaml

```diff
@@ -26,10 +26,7 @@ deploy:
         wait: true
         remoteChart: opentelemetry-operator
         repo: https://open-telemetry.github.io/opentelemetry-helm-charts
-        setValues:
-          admissionWebhooks.certManager.enabled: false # Disable certManager
-          admissionWebhooks.certManager.autoGenerateCert.enabled: true # Helm will create a self-signed cert and secret
-          image.pullPolicy: "IfNotPresent"
+        valuesFiles: ["./helm_values.yaml"]
```

Comment:
Was this just aesthetic or is there some reason we needed to change it?

Reply:
I left a note in the yaml file, it's because skaffold `setValues` doesn't work with the limitation of no "additionalProperties" in the new version of opentelementry. 

# The setValues are here instead of in the skaffold.yaml file because as of opentelemetry-operator
# version 0.50.0 there must be a --set flag for each key-value pair. The current behavior in
# skaffold `setValues` is to append all key-pair pairs to a single --set flag which results in an
# `Error: opentelemetry-operator: - (root): Additional property image is not allowed`.

Reply:
Oh, I read past your note because I thought it was all boilerplate.

Also... weird. But whatever works tbh.